docs/on-prem.rst

```diff
@@ -1432,7 +1432,7 @@ Before running this section:
 
    .. sourcecode:: bash
 
-      docker-compose exec kafka1 kafka-replica-status \
+      docker exec kafka1 kafka-replica-status \
```

Comment:
@chuck-confluent it's post merge but why was this line changed?

Reply:
I played with it a bit (probably even more than I should). The behavior of docker exec is different than docker-compose exec. We set the container name, so it's equivalent to docker-compose exec except the output pipes into grep more nicely.